test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs

```diff
@@ -157,14 +157,8 @@ public void AppHostShowsErrorWhenCurrentTargetFrameworkWasNotFoundInProjectJson(
                     stdOut: out stdOut,
                     stdErr: out stdErr);
 
-                var expectedErrorMsg =$@"The current runtime target framework is not compatible with '{projectName}'.
-
-Current runtime Target Framework: '{runtimeTargetFramework} ({runtimeTargetFrameworkShortName})'
-  Type: {runtimeType}
-  Architecture: {architecture ?? TestUtils.CurrentRuntimeEnvironment.RuntimeArchitecture}
-  Version: {TestUtils.GetRuntimeVersion()}
-
-Please make sure the runtime matches a framework specified in {Project.ProjectFileName}";
+                // TODO: add complete error message check when OS version information is consistent on CoreCLR and CLR
```

Comment:
On my Windows 8.1, `RuntimeEnvironment.OperatingSystemVersion` returns different values for dnx451 and dnxcore50
For dnx451, it returns `6.2.9200.0`
For dnxcore50, it returns `6.2`